### DISCUSSION TITLE
Is it possible to override from a list to a set for a POJO that is generated from a schema with the list type

### DISCUSSION THREAD

**Discussion by siva54** [kobylynskyi/graphql-java-codegen#917] [Q&A]
Is it possible to override from a list to a set for a POJO that is generated from a schema with the list type?

I have a schema definition, 
versions: [String!]

Instead of using a List in the POJO like this,
List<String> versions

I want to use a set like this,
Set<String> versions

Is this possible?

**Comment by kobylynskyi** [comment]
Right now that is not possible. Please create a feature request.

What you can do in the current version is to specify the response type for each list field individually using customTypeMapping

**Reply by siva54** [reply]
Thank you @kobylynskyi . Can you suggest an example perhaps? That will help me.
